sys/docker/alpine-nginx/Dockerfile

```diff
@@ -14,12 +14,12 @@ RUN apk add --no-cache --virtual frontenddeps \
     && yarn add sass-loader@^8.0.0 node-sass --dev \
     && npm run build
 
-FROM fabiocicerchia/nginx-lua:1.19.3-alpine3.12.0
+FROM fabiocicerchia/nginx-lua:1.19.3-alpine3.12.1
 
 WORKDIR /application
 
 COPY sys/nginx/nginx.conf /etc/nginx/nginx.conf
-COPY sys/nginx/default.conf /etc/nginx/conf.d/default.conf
+COPY sys/nginx/default.conf.template /etc/nginx/templates/default.conf.template
```

Comment:
Who use this file? .. where is the hidden magic?

Reply:
https://github.com/fabiocicerchia/nginx-lua#using-environment-variables-in-nginx-configuration-new-in-119

It's a behavior of the official nginx image (which I've replicated in the lua mod) using envsubst to replace the env vars in the template and move it in the conf.d folder.

Reply:
uh ok .. I didn't know about it. tnx